riscv-sbi.adoc

```diff
@@ -0,0 +1,174 @@
+# RISC-V Supervisor Binary Interface Specification
+
+## Copyright and license information
+
+This RISC-V SBI specification is
+
+[%hardbreaks]
+(C) 2019 Palmer Dabbelt <palmer@sifive.com>
+(C) 2019 Atish Parta <atish.parta@wdc.com>
+
+It is licensed under the Creative Commons Attribution 4.0 International
+License (CC-BY 4.0).  The full license text is available at
+https://creativecommons.org/licenses/by/4.0/.
+
+## Introduction
+
+This specification describes the RISC-V Supervisor Binary Interface, known from
+here on as SBI.  This interface allows supervisor-mode software to be written
+that is portable to all RISC-V implementations.  The design of the SBI follows
+the general RISC-V philosophy of having a small core along with a set of
+optional modular extensions.
+
+### Versioning and History
+
+This document describes a draft of version 0.2 of the RISC-V SBI specification.
+
+#### Changes Since Version 0.1
+
+* The entire v0.1 SBI has been moved to the legacy extension, which is now an
+  optional extension.  This is technically a backwards-incompatible change, 
```

Comment:
Lets keep the legacy function descriptions in 0.2 legacy section as the next patch is removing the 0.1 document completely.